### PR TITLE
Add OMEMO.

### LINKED ISSUE #1476: Add OMEMO to /protocols/omemo
So far, two software support [OMEMO](http://conversations.im/omemo/) encryption, [Conversations](http://conversations.im/), [Cryptocat](https://crypto.cat/) and [Gajim](https://gajim.org/) (as plugin).

I suggest to add `/protocols/omemo` to raise awareness and receive more feedbacks on OMEMO, and to instigate its support on other software platforms.

### PR DESCRIPTION
**Pull request by mimi89999**
Fixes #1476

### CODE REVIEW

**Comment by mimi89999** [comment]
@nylira Could you also check this PR?

**Comment by strugee** [comment]
@mimi89999 sorry, didn't see this (this was before I became comaintainer). I'll review.

**Comment by mimi89999** [comment]
@strugee I will update it adding it to apps supporting it.

File: source/db/en-projects.json [hunk 1]
```diff
@@ -889,6 +889,7 @@
     "wikipedia_url": "",
     "protocols": [
       "XMPP",
+      "OMEMO",
```
Comment (strugee) [inline source/db/en-projects.json]: Pretty sure there will be outdated English localizations in other language files. You'll need to adjust those too or (even better) just remove them entirely.

File: source/db/protocols/en-protocols.json [hunk 2]
```diff
@@ -72,6 +72,15 @@
     }]
   },
   {
+    "name": "OMEMO",
```
Comment (strugee) [inline source/db/protocols/en-protocols.json]: Unfortunately localized protocol definitions don't inherit from the English version like projects do, so you'll need to make this same change to all other protocol JSON files. Sorry :(

I'd suggest writing a `for` loop in the shell with `patch` or something.

**Comment by mimi89999** [comment]
@strugee Done

**Comment by mimi89999** [comment]
@strugee So it's ready now.

File: source/db/en-projects.json [hunk 3]
```diff
@@ -562,17 +562,18 @@
   },
   {
     "development_stage": "released",
-    "description": "OTR-encrypted IM for Android and iOS.",
+    "description": "OMEMO or OTR encrypted IM for iOS.",
     "license_url": "https://chatsecure.org/developers/",
     "logo": "chatsecure.png",
-    "notes": "The Guardian Project hosts <a href='https://guardianproject.info/howto/chatsecurely/'>a fantastic how-to guide</a> to chatting securely on Android.",
```
Comment (strugee) [inline source/db/en-projects.json]: Why is this being removed?

File: source/db/protocols/eo-protocols.json [hunk 4]
```diff
@@ -18,6 +18,15 @@
     }]
   },
   {
+    "name": "Matrix",
```
Comment (strugee) [inline source/db/protocols/eo-protocols.json]: This seems to have been accidentally introduced; please remove

File: source/db/en-projects.json [hunk 5]
```diff
@@ -562,17 +562,18 @@
   },
   {
     "development_stage": "released",
-    "description": "OTR-encrypted IM for Android and iOS.",
```
Comment (strugee) [inline source/db/en-projects.json]: I think this looks better:

> OMEMO- or OTR-encrypted IM for iOS

but feel free to disagree. Also, please remove now-outdated localizations from other languages.

Comment (mimi89999) [inline source/db/en-projects.json]: I don't see a reason to put a dash there...

File: source/db/en-projects.json (hunk 3 above)
Comment (mimi89999) [inline source/db/en-projects.json]: Note explaining how to install ChatSecure on Android. Forgot it in #1710

File: source/db/en-projects.json (hunk 5 above)
Comment (strugee) [inline source/db/en-projects.json]: @mimi89999 I believe that's proper English, but I don't feel strongly about it.

File: source/db/en-projects.json (hunk 3 above)
Comment (strugee) [inline source/db/en-projects.json]: @mimi89999 please split that up into a separate commit, then. Thanks :)

File: source/db/protocols/eo-protocols.json (hunk 4 above)
Comment (mimi89999) [inline source/db/protocols/eo-protocols.json]: It was missing and the file was to short for my script to put OMEMO in the correct place.

**Comment by mimi89999** [comment]
@strugee Changed. Can you review again?

File: source/db/protocols/eo-protocols.json (hunk 4 above)
Comment (strugee) [inline source/db/protocols/eo-protocols.json]: This didn't get addressed; please remove it. It'll be inherited from the English version, remember.

File: source/db/ru-projects.json [hunk 6]
```diff
@@ -562,17 +562,18 @@
   },
   {
     "development_stage": "released",
-    "description": "IM-клиент с поддержкой шифрования для iOS и Android.",
+    "description": "OMEMO or OTR encrypted IM for iOS.",
     "license_url": "https://chatsecure.org/developers/",
     "logo": "chatsecure.png",
-    "notes": "The Guardian Project предоставляет <a href='https://guardianproject.info/howto/chatsecurely/'>фантастическую инструкцию</a> о том, как общаться безопасно на Android.",
```
Comment (strugee) [inline source/db/ru-projects.json]: This seems to have been rewritten accidentally.

**Comment by mimi89999** [comment]
@strugee Corrected. Thanks.

File: source/db/de-projects.json [hunk 7]
```diff
@@ -562,9 +562,10 @@
   },
   {
     "development_stage": "released",
-    "description": "Verschlüsselte Sofortnachrichten für iOS.",
+    "description": "OMEMO or OTR encrypted IM for iOS.",
     "license_url": "https://chatsecure.org/developers/",
     "logo": "chatsecure.png",
+    "notes": "Das Guardian Project hosted <a href='https://guardianproject.info/howto/chatsecurely/'>eine fantastische Anleitung</a> zum si",
```
Comment (strugee) [inline source/db/de-projects.json]: This seems to have been added accidentally

File: source/db/zh-CN-projects.json [hunk 8]
```diff
@@ -29,13 +29,13 @@
   },
   {
     "development_stage": "experimental",
-    "description": "基于开源安卓项目，帮助用户检测和避开伪基站（IMSI 捕捉器）或其他无加密/弱加密基站。本项目旨在协助用户了解加密和其他一些保护机制是否处于启用状态。事物总是在发展，我们永远欢迎测试者以及热衷信息安全的开发者。不必拘束，你可以做任何你能做的贡献。我们的 [GitHub](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector)。我们的 [XDA](https://forum.xda-developers.com/showthread.php?t=1422969)。感谢你的参与！",
+    "description": "基于开源安卓项目，帮助用户检测和避开伪基站（IMSI捕捉器）或其他无加密/弱加密基站。本项目旨在协助用户了解加密和其他一些保护机制是否处于启用状态。事物总是在发展，我们永远欢迎测试志愿者以及关注信息安全的程序员志士。不必害羞，你可以做任何你能做的贡献。我们的Github：https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector。我们的XDA：https://forum.xda-developers.com/showthread.php?t=1422969。谢谢你们的参与！",
```
Comment (strugee) [inline source/db/zh-CN-projects.json]: This entire file seems to have gotten mangled. An update to the Chinese localizations got merged; maybe you just need to rebase or something?

File: source/db/zh-CN-projects.json [hunk 9]
```diff
@@ -1287,32 +1289,7 @@
   },
   {
     "development_stage": "released",
-    "description": "私密的 XMPP 服务，运行在 DuckDuckGo 社区平台。&mdash;包含使用说明。",
```
Comment (strugee) [inline source/db/zh-CN-projects.json]: This definitely should be gone, but should be done in a separate commit. https://www.freshconsulting.com/atomic-commits/

Not sure if you did this on purpose.

File: source/db/zh-CN-projects.json (hunk 8 above)
Comment (mimi89999) [inline source/db/zh-CN-projects.json]: @strugee Done.

File: source/db/zh-CN-projects.json (hunk 9 above)
Comment (mimi89999) [inline source/db/zh-CN-projects.json]: @strugee Not my fault here.
https://github.com/nylira/prism-break/commit/3a748b1741e85d1f9968c96c2c6aa584d2eab557

**Comment by mimi89999** [comment]
@strugee Is it OK now?

File: source/db/de-projects.json (hunk 7 above)
Comment (strugee) [inline source/db/de-projects.json]: @mimi89999 this didn't get addressed; otherwise LGTM

Comment (mimi89999) [inline source/db/de-projects.json]: @strugee Corrected and sorry.

Comment (strugee) [inline source/db/de-projects.json]: @mimi89999 no problem! Thanks for being patient :)
